test/jdk/java/net/InetAddress/OfLiteralTest.java

```diff
@@ -22,7 +22,7 @@
  */
 
 /* @test
- * @bug 8272215
+ * @bug 8272215 8315767
  * @summary Test for ofLiteral API in InetAddress classes
```

Comment:
The test summary can mention that `ofLiteralPosix` is also tested here

Reply:
done